### DISCUSSION TITLE
Mini comment seems to be moved to extras

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#3195] [General]
Mini comment seems to be used to extras but why are we still including by default
https://github.com/LazyVim/LazyVim/blob/6aef1989bd1c08a828eecb7e9044bf8cc1d93700/lua/lazyvim/plugins/coding.lua#L193C1-L202C8

**Comment by folke** [comment]
That code you point to does exactly that.

The `mini.comment` import will only be `enabled()` when not on `0.10.0`.
That extra code is to make `ts-context-commentstring` work with native comments.

So to summarize, on 0.10, we use native comments and on older versions still mini.comment

**Reply by rubiin** [reply]
Thankyou for the clarification :)
